kura/org.eclipse.kura.deployment.agent/src/main/java/org/eclipse/kura/deployment/agent/DeploymentAgentService.java

```diff
@@ -99,6 +99,18 @@ public interface DeploymentAgentService {
      */
     public boolean isInstallingDeploymentPackage(String url);
 
+
+    /**
+     * Provides the Eclipse Marketplace Package Descriptor information of the deployment package identified by URL
+     * passed as query parameter.
+     *
```

Comment:
Since this is a new method in an exported package and it has a javadoc, please also add `@since 1.4.0` annotation. (1.4.0 is the bundle version that introduces the new method)

Reply:
Addressed in 280d140e1ec4442e1c89dfeea17c83967b651184